src/main/scala/cromwell/backend/model/ExecutionResult.scala

```diff
@@ -32,5 +32,4 @@ case class FailureResult(exception: Throwable) extends ExecutionResult
   * @param overallHash Contains a hash from TaskDescriptor.
   * @param dockerHash Contains docker image hash.
   */
-// TODO: remove dockerHash.
-case class ExecutionHash(overallHash: String, dockerHash: Option[String])
\ No newline at end of file
+case class ExecutionHash(overallHash: String, dockerHash: Option[String] = None)
```

Comment:
Yes, the value of dockerHash factors into the value of overallHash.  But we want to record the Docker hash to have the provenance of which Docker image was actually used to execute a command.